docs/spec/Changelog.schema.json

```diff
@@ -16,9 +16,12 @@
       },
       "from_file": {
         "type": "boolean"
+      },
+      "public": {
```

Comment:
I'm now not sure how I discovered this unrelated bug, but this schema was wrong - tests were passing because trivially because they were querying a report with no changelogs. Have updated this test: https://github.com/vimc/orderly-web/pull/488/files#diff-deddbcec100ca7872325eb767526818006a61b7bd521ceea1a66c85809c9be52R118